packages/java/tests/pom.xml

```diff
@@ -158,6 +163,9 @@
                     <groupId>com.vaadin</groupId>
                     <artifactId>hilla-maven-plugin</artifactId>
                     <version>${project.version}</version>
+                    <configuration>
+                        <generateBundle>false</generateBundle>
```

Comment:
While technically not required, this reduces the noise in the untracked files caused by creating the `prod.bundle` in ITs.